Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ForegroundRiskCalculationService.kt

```diff
@@ -75,7 +92,8 @@ class WatchdogService @Inject constructor(
         .apply { acquire() }
 
     companion object {
-        private const val TAG = "WatchdogService"
+        private const val TAG = "ForegroundRiskCalc"
         private const val TEN_MINUTE_TIMEOUT_IN_MS = 10 * 60 * 1000L
+        private val minTimeBetweenRuns = Duration.ofMinutes(60)
```

Comment:
could this cause an issue with ENF quota which is 6 times per day ? should we increase this duration

Reply:
no, that is checked in the task itself